test/mtz-file-download-behavior_test.html

```diff
@@ -14,15 +14,93 @@
   <body>
     <test-fixture id="basic">
       <template>
-        <mtz-file-download-behavior></mtz-file-download-behavior>
+        <file-downloader></file-downloader>
       </template>
     </test-fixture>
 
+    <dom-module id="file-downloader">
+      <template>
+        <button on-click="openUriInNewTab">Open In New Tab</button>
+        <button on-click="openUriInCurrentTab">Open In Current Tab</button>
+        <button on-click="createFile">Create File</button>
+      </template>
+      <script>
+        window.addEventListener('WebComponentsReady', () => {
+          Polymer({
+            is: 'file-downloader',
+            behaviors: [
+              // Register the behavior
+              mtz.FileDownloadBehavior
+            ],
+            // Open data at a URI in a new tab or the current tab allowing the user
+            // to save if they so desire.
+            openUriInNewTab() {
+              this.downloadFromURI('./sample.txt', true);
+            },
+            openUriInCurrentTab() {
+              this.downloadFromURI('./sample.txt', false);
+            },
+            // Serialize a string of data to a <mime-type> file.
+            createFile() {
+              const data = "You've serialized data to file. Happy days!!!";
```

Comment:
I'm encountering an issue with CORS related to opening the csv in the same window as that of the demo. Any recommendations on how to circumvent that issue?